src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -49,22 +49,32 @@ interface BooleanDictionary {
 const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = ({
   data,
   isFetching,
-  user
+  userPermissions
 }) => {
-  const [columns] = useState([
-    { title: 'Name', transforms: [sortable], cellFormatters: [expandable] },
-    { title: 'Version', transforms: [sortable] },
-    {
-      title: (
-        <>
-          Simple Content Access
-          <SCAInfoIconWithPopover />
-        </>
-      ),
-      transforms: [sortable, cellWidth(20)]
-    },
-    { title: 'UUID', transforms: [sortable] }
-  ]);
+  const getColumns = (userPermissions: UserPermissions) => {
+    const columns = [
+      { title: 'Name', transforms: [sortable], cellFormatters: [expandable] },
+      { title: 'Version', transforms: [sortable] },
+      {
+        title: (
+          <>
+            Simple Content Access
+            <SCAInfoIconWithPopover />
+          </>
+        ),
+        transforms: [sortable, cellWidth(20)]
+      },
+      { title: 'UUID', transforms: [sortable] }
+    ];
+
+    if (userPermissions.isSCACapable === false) {
+      // remove SCA Status column
+      columns.splice(2, 1);
+    }
+    return columns;
+  };
+
+  const columns = getColumns(userPermissions);
```

Comment:
Can you just call the getColumns function directly in the `cells={columns}` location below instead of having this extraneous variable?  Then you could move the function down below all this state stuff with the other functions.

Maybe we should also rename the getColumns function because it's not very clear that it's just metadata about the columns.  Of course the prop it's being passed to in Table also isn't clear, but there's nothing to do about that.

Reply:
OK I have it called now directly. Also I renamed it to `getManifestPanelColumns`. Not sure if that's the best name; was having trouble thinking of a better one. LMK if you have any thoughts.

Reply:
I was thinking something like getColumnInfo or getColumnSpecifications, just to convey that it returns information about columns, but not actually the data that will be in the columns.  Also could you move the function down to where the other functions are, below the useState calls and regular variable declarations?